biscuit/core/utils/events.py

```diff
@@ -57,7 +57,10 @@ def quit(self, *_):
         self.base.destroy()
     
     def toggle_maximize(self, *_):
-        self.base.wm_state('normal' if self.maximized else 'zoomed')
+        if self.base.sysinfo.os == "Windows":
```

Comment:
```suggestion
        if self.base.sysinfo.os != "Linux":
```

This way it supports macOS as well.